dtls.h

```diff
@@ -303,8 +320,11 @@ int dtls_renegotiate(dtls_context_t *ctx, const session_t *dst);
  * @return The number of bytes written, @c -1 on error or @c 0
  *         if the peer already exists but is not connected yet.
  */
-int dtls_write(struct dtls_context_t *ctx, session_t *session, 
-	       uint8 *buf, size_t len);
+static inline
+int dtls_write(struct dtls_context_t *ctx, session_t *session,
+	       uint8 *buf, size_t len) {
+  return dtls_write_multi(ctx, session, &buf, &len, 1);
+}
```

Comment:
@obgm 

I assumed, this is the function for backwards compatibility.
Does it work for your?

Reply:
Yes, sorry. I got confused by the diff output.